sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/RequestTimeline.java

```diff
@@ -148,25 +149,33 @@ public String toString() {
         return RntbdObjectMapper.toString(this);
     }
 
-    @JsonPropertyOrder({ "name", "time", "duration" })
+    @JsonPropertyOrder({ "name", "startTime", "durationInMicroSec" })
     public static final class Event {
 
-        @JsonSerialize(using = ToStringSerializer.class)
+        @JsonIgnore
         private final Duration duration;
 
+        @JsonSerialize(using = ToStringSerializer.class)
+        private final long durationInMicroSec;
```

Comment:
I wonder why we have mixture of Duration and Long for representing time.
Is there a specific reason we are not using the same type?

Reply:
Reason is  these number will be so small,  i want to show it in micro sec , and didn't want to introduce new serializer class for this one variable , however i am good with anything , please let me know.